pegasus/sites.v3/code.org/views/top_hoc_tutorials.haml

```diff
@@ -1,9 +1,14 @@
 = view :top_hoc_tutorial_applab, image_match_youtube_height: false
 
+- if ['mc', 'dance'].include? DCDO.get('hoc_launch', nil)
+  - minecraft_img_url = CDO.code_org_url("/shared/images/courses/logo_tall_aquatic.jpg")
```

Comment:
The source image is 1160px wide, so should we include a resize (e.g. `fill-`) in this URL as well?

Reply:
yeah, that's a good call. looking at the image on line 6, i'm not sure i understand the resize rules for a URL, though. are we using a gem for this? or is there documentation around how resizing works? 

not sure if this matters, but it may be worth pointing out that the image on line 6 is stored in `pegasus/` and the image here is stored in `shared/`

Reply:
It's a little internal pegasus service (implemented [here](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/pegasus/graphics.rb) and tested [here](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/test/test_graphics.rb)).  It should work for images in `shared/` as well.

Reply:
awesome, thank you! i'll make that change now